client/src/utils/fetchUserData.js

```diff
@@ -46,6 +47,22 @@ export async function fetchProfileData(user) {
     return response;
   } catch (err) {
     console.error(err);
+    fetchLogOut();
+  }
+}
+
+export async function fetchImageToAI(image) {
+  try {
+    const formData = new FormData();
+    formData.append('image', image);
+    const res = await fetch('https://64ee-2601-646-c600-3560-d419-a27c-984f-1c5.ngrok-free.app/upload', {
```

Comment:
is this an api key that should be an env var ?

Reply:
That is a good call. I have moved it to an env file.

This is a temporary key that is going to a local machine and we will change the link once it is deployed.